flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/config/TableConfigOptions.java

```diff
@@ -94,8 +94,16 @@ private TableConfigOptions() {}
     public static final ConfigOption<Integer> MAX_LENGTH_GENERATED_CODE =
             key("table.generated-code.max-length")
                     .intType()
-                    .defaultValue(64000)
+                    .defaultValue(4000)
                     .withDescription(
                             "Specifies a threshold where generated code will be split into sub-function calls. "
                                     + "Java has a maximum method length of 64 KB. This setting allows for finer granularity if necessary.");
+
+    @Documentation.TableOption(execMode = Documentation.ExecMode.BATCH_STREAMING)
+    public static final ConfigOption<Integer> MAX_MEMBERS_GENERATED_CODE =
```

Comment:
Can this one be internal config option? I don't think it is a frequently-used option.